index.html

```diff
@@ -107,77 +109,82 @@ <h4 class="works__single__title">Multi-Post Stories</h4>
           alt="project 1"
         />
 
-        <h4 class="works__single__title">Tonic</h4>
+        <div class="works__single__holder">
+          <h4 class="works__single__title">Tonic</h4>
 
-        <div class="works__single__details">
-          <p
-            class="works__single__details__single works__single__details__single--dark"
-          >
-            CANPOY
-          </p>
-          <div class="dot"></div>
-          <p
-            class="works__single__details__single works__single__details__single--light"
-          >
-            Back End Dev
-          </p>
-          <div class="dot"></div>
-          <p
-            class="works__single__details__single works__single__details__single--light"
-          >
-            2015
+          <div class="works__single__details">
+            <p
+              class="works__single__details__single works__single__details__single--dark"
+            >
+              CANPOY
+            </p>
+            <div class="dot"></div>
+            <p
+              class="works__single__details__single works__single__details__single--light"
+            >
+              Back End Dev
+            </p>
+            <div class="dot"></div>
+            <p
+              class="works__single__details__single works__single__details__single--light"
+            >
+              2015
+            </p>
+          </div>
```

Comment:
- [ ] Consider using the correct font size for these elements. I suggest you check the Figma template to get the correct font size and apply them to the `CSS`